contracts/contracts/multisig/Multisig.cairo

```diff
@@ -383,23 +371,21 @@ func submit_transaction{
 end
 
 @external
-func confirm_transaction{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(tx_index : felt):
+func confirm_transaction{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    tx_index : felt
+):
     require_owner()
     require_tx_exists(tx_index=tx_index)
+    require_tx_valid(tx_index=tx_index)
     require_not_executed(tx_index=tx_index)
+    # TODO: clarify if needed
```

Comment:
This TODO can be removed, right?

Reply:
I think @krebernisak means keep the code but remove the `TODO` comment only